src/Honeybee_Add to EnergyPlus Library.py

```diff
@@ -50,7 +50,32 @@
 
 import scriptcontext as sc
 import Grasshopper.Kernel as gh
+import os
 
+def prepEPObjects(rawStrings):
```

Comment:
@mostaphaRoudsari , where is importEPLibraryFromFile? in Honeybee_Honeybee? I couldn't find it. What do you mean not clean,  like idf file with "!" notes? Does this method returns EPObjects?